cmd/ak/common/flags.go

```diff
@@ -33,27 +34,31 @@ func FailNotFound(cmd *cobra.Command, what string) error {
 	return nil
 }
 
-func ToExitCodeError(err error, what string) error {
+func ToExitCodeError(err error, whats ...string) error {
 	if err == nil {
 		return nil
 	}
-	msg := what
+	msg := strings.Join(whats, " ")
 	var code int = GenericFailure
+
 	switch {
 	case errors.Is(err, sdkerrors.ErrNotFound):
-		return NewExitCodeError(NotFoundExitCode, fmt.Errorf("%s not found", what))
+		return NewExitCodeError(NotFoundExitCode, fmt.Errorf("%s not found", msg))
```

Comment:
```suggestion
		return NewExitCodeError(NotFoundExitCode, fmt.Errorf("%q not found", msg))
```

and elsewhere

Reply:
yes, @daabr is right. That's the whole message which IMO shouldn't be formatted as %q